lib/v2/screens/receive/components/receive_selection_card.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+
+class ReceiveSelectionCard extends StatelessWidget {
+  final String title;
+  final Widget? icon;
+  final GestureTapCallback? callback;
+
+  const ReceiveSelectionCard({
+    Key? key,
+    required this.title,
+    this.icon,
+    required this.callback,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      borderRadius: BorderRadius.circular(defaultCardBorderRadius),
+      onTap: callback,
+      child: Ink(
+        decoration: BoxDecoration(
+          color: AppColors.darkGreen2,
+          borderRadius: BorderRadius.circular(defaultCardBorderRadius),
+        ),
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            icon ?? const SizedBox.shrink(),
```

Comment:
this is not needd once the icon is not nullable

Reply:
done